### ISSUE TITLE
WCAG 2.1 Compliance

### ISSUE THREAD

**Issue by mgifford** [wet-boew/wet-boew#8602]
There are only 2 issues tagged with WCAG 2.1 here:

https://github.com/wet-boew/wet-boew/issues?q=is%3Aissue+wcag+2.1+label%3A%22Work%3A+WCAG+2.1%22

Does this mean that there hasn't been a review against the WCAG 2.1 guidelines or that there has been done and we're mostly good?  If the former how do we move ahead so that future WxT releases are WCAG 2.1 compliant?

**Comment by juleskuehn** [comment]
AFAIK there has been no attempt to review for WCAG 2.1.

**Comment by duboisp** [comment]
The project hasn't been review against the WCAG 2.1 guidelines yet. 

I know there is an interest to meet WCAG 2.1 Level AA for future major release of WET, like with WET5.

@mgifford are you available to help or lead a such project?

**Comment by mgifford** [comment]
Hey @duboisp I could lead the 2.1 upgrade, but it would have to come with a contract. It's a lot of work to do the review & testing (even to just see it still meets WCAG 2.0 AA).

**Comment by bsouster** [comment]
I will make this my focus when I have time to put in. I'll start with getting people testing the template pages

**Comment by mgifford** [comment]
Any update @bsouster?

**Comment by bsouster** [comment]
@mgifford Just got a bunch of students -- it's in my calendar for this month

Suggestions where to prioritize we start testing is appreciated (+ @duboisp)

**Comment by mgifford** [comment]
Generally I figure start small with the easy wins and go from there. Trouble is what are those....  Assessing WCAG 2.1 is getting easier, but still not trivial. Requires and understanding of WCAG 2.0 to understand what is missing. 

I recommend starting with looking at Drupal though https://www.drupal.org/project/issues/search?issue_tags=wcag21

Not that this is comprehensive, but at least this is a list where you can start to see some of the issues we're looking at.
